website/scripts/build-static-content.js

```diff
@@ -354,11 +354,15 @@ module.exports = {
               // >   <meta name="title" value="Sth with punctuATION and weird CAPS ... but never this long, please">
               // >   ```
               let embeddedMetadata = {};
-              for (let tag of (mdString.match(/<meta[^>]*>/igm)||[])) {
-                let name = tag.match(/name="([^">]+)"/i)[1];
-                let value = tag.match(/value="([^">]+)"/i)[1];
-                embeddedMetadata[name] = value;
-              }//∞
+              try {
+                for (let tag of (mdString.match(/<meta[^>]*>/igm)||[])) {
+                  let name = tag.match(/name="([^">]+)"/i)[1];
+                  let value = tag.match(/value="([^">]+)"/i)[1];
+                  embeddedMetadata[name] = value;
+                }//∞
+              } catch (err) {
+                throw new Error('An error occured while parsing <meta> tags in Markdown. Tip: Check the markdown being changed and make sure it doesn\'t contain any code snippets with <meta> inside, as this can fool the build script. Full error: '+err);
```

Comment:
Two things here:
1. Rather than doing `+err`, which will try to call the Error instance's toString() function (which often isn't very helpful), it's better to do `+err.stack`, which will include the message and a relevant stack trace.  If you ever prefer not to include the trace, use `+err.message`.
2. Should we make this match the error messages from elsewhere in this file?  Both for consistency, and because they include the path of the offending file, which is handy when debugging:
<img width="452" alt="image" src="https://user-images.githubusercontent.com/618009/191159232-e6ace436-853c-420c-acac-657e0c96c3f4.png">

Reply:
Makes sense! I've updated the error message with those changes.